app/views/shared/_comment.html.erb

```diff
@@ -3,7 +3,12 @@
 <% else %>
   <% post.most_recent_comments.each do |comment| %>
     <ul>
-      <li><%= comment.author.name %>: <%= comment.text %></li>
-    </ul>
+      <li><%= comment.author.name %>: <%= comment.text %></li> 
```

Comment:
I will leave the comment here since you do not have any changes on the comments/new.html.erb file.

### Add new comment authorization
Kindly make sure that the current/logged-in user is authorized and able to comment on any post. Currently, I am not able to add comments to posts.

![image](https://user-images.githubusercontent.com/38088818/232017193-61be5ccc-936e-495a-8b13-54b9c33cb92b.png)

- Since I cannot create a new comment, I'm unable to verify if the author of the comment can delete the newly created comment. When this issue is resolved, kindly make sure you check and make sure that the owner can delete their comment.